### DISCUSSION TITLE
Delete items from DB and overwrite  it

### DISCUSSION THREAD

**Discussion by botzill** [directus/directus#2879] [Archive]
Currently, if I remove an item from a collection that has a unique field, I can't create another item in that collection with the same unique field. 

Is it possible to completely remove some items from the collection without manually clear them from DB?

**Comment by rijkvanzanten** [comment]
Sounds like you have soft-delete configured for this collection. Removing the status field that has the soft-delete value from the collection will make sure everything gets fully deleted when you hit delete

**Comment by botzill** [comment]
I see. So with the status field, there is no way to as well delete them permanent?  Do I need to create another field that will mimic status and use that instead?
